storage/scylla_api.c

```diff
@@ -6,7 +6,7 @@
  * "LICENSE" at the root of this distribution.
  */
 #include "scylla_api.h"
-
+#include <bsd/string.h>
```

Comment:
Why did you invoke BSD-specific functions?

Reply:
To use strlcat instead of strcat and strncat.
I wondered about invoking BSD library too.
Is there a better method?

Reply:
Reference: https://lwn.net/Articles/612244/

Reply:
After read that, I would like to remove BSD library and concentrate string myself. Actually, it's not so complicated here.